kombu/asynchronous/aws/sqs/connection.py

```diff
@@ -20,18 +20,26 @@
 class AsyncSQSConnection(AsyncAWSQueryConnection):
     """Async SQS Connection."""
 
-    def __init__(self, sqs_connection, debug=0, region=None, fetch_message_attributes=None, **kwargs):
+    def __init__(
+        self,
+        sqs_connection,
+        debug=0,
+        region=None,
+        message_system_attribute_names=None,
+        message_attribute_names=None,
```

Comment:
as we are removing fetch_message_attributes=None, this might break BC? or not?

Reply:
I do not believe so, but Kombu does have a lot going on. I updated where we instantiate `AsyncSQSConnection`
``` 
AsyncSQSConnection(
    sqs_connection=self.sqs(queue=queue),
    region=self.region,
    message_system_attribute_names=message_system_attribute_names,
    message_attribute_names=message_attribute_names
)
```

I think we have decent test coverage for when no values are provided in addition to various values being provided.